src/st_paywall/stripe_auth.py

```diff
@@ -33,22 +34,29 @@ def redirect_button(
     else:
         raise ValueError("payment_provider must be 'stripe' or 'bmac'")
 
-    st.sidebar.markdown(
-        f"""
+    button_html = f"""
     <a href="{button_url}" target="_blank">
         <div style="
             display: inline-block;
             padding: 0.5em 1em;
             color: #FFFFFF;
             background-color: {color};
-            border-radius: 3px;
-            text-decoration: none;">
+            border-radius: 0.25rem;
+            border: 1px solid rgba(49, 51, 63, 0.2);
+            text-decoration: none;
+            text-align: center;
+            cursor: pointer;
+            font-weight: 400;
+            box-shadow: rgba(0, 0, 0, 0.08) 0px 1px 2px;">
             {text}
         </div>
     </a>
-    """,
-        unsafe_allow_html=True,
-    )
+    """
+    
+    if use_sidebar:
+        st.sidebar.markdown(button_html, unsafe_allow_html=True)
```

Comment:
Could move to st.html and limit streamlit versions a bit

Reply:
done!